Projects/Calendar/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<!-- Coding By CodingNepal - youtube.com/codingnepal -->
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8" />
+    <title>Dynamic Calendar JavaScript | CodingNepal</title>
+    <link rel="stylesheet" href="styles.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Google Font Link for Icons -->
+    <link
```

Comment:
@khushigarg1 Could you explain this? This project seems to have be copied.
Link : https://www.youtube.com/watch?v=Z1BGAivZRlE

Reply:
sorry i just took a html file from there but css and all and some animations that i added myself.

Reply:
@khushigarg1 It's okay to refer to YouTube videos for your projects but make sure you make the final project on your own. Best of Luck ✨

Reply:
> @khushigarg1 Could you explain this? This project seems to be copied from the below mentioned youtube channel. Link : https://www.youtube.com/watch?v=Z1BGAivZRlE

Yes thank you!